static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -168,6 +168,8 @@ except in mobile */
   display: flex;
   position: relative;
 
+  height: ${p.theme.form.md.height};
```

Comment:
They shouldn't ever get as large as in the initial screenshot, so I wonder if there might be some custom CSS rule that could be removed? In theory, if that is fixed, I would expect the size to drop down to the size of the rest of the buttons in that row, and so this height might not even be needed